ecommerce/serializers.py

```diff
@@ -172,10 +172,14 @@ def get_discounted_price(self, instance):
         discounts = instance.discounts.all()
 
         if discounts.count() == 0:
-            return None
+            return sum(
+                (basket_item.quantity * basket_item.product.price)
+                for basket_item in instance.basket_items.all()
+            )
 
         return sum(
-            basket_item.discounted_price for basket_item in instance.basket_items.all()
+            (basket_item.quantity * basket_item.discounted_price)
```

Comment:
I think discounted_price is already price*quantity. I am looking here: https://github.com/mitodl/mitxonline/blob/main/ecommerce/models.py#L83

Reply:
Think I missed something in the last commit.. should be fixed now though.